src/components/reusable/table/table-header.ts

```diff
@@ -157,8 +179,21 @@ export class TableHeader extends LitElement {
     );
   }
 
-  override updated() {
+  override updated(changedProperties: PropertyValues) {
     this.getTextContent();
+
+    super.updated(changedProperties);
+    if (this.maxWidth && changedProperties.has('maxWidth')) {
+      this.style.setProperty('--kyn-th-max-width', this.maxWidth);
+    }
+
```

Comment:
IMO no need to check `this.maxWidth` here as it's already covered in `changedProperties.has('maxWidth')`. So I guess your if condition will be 

`if(changedProperties.has('maxWidth'){ ... }`

Same applies for other if conditions also. [`this.minWidth` and `this.width`]

Reply:
changedProperties.has('maxWidth') only checks if there is a change of maxWidth, meanwhile this.maxWidth is checking for a truthy (i.e., it has a value other than undefined, null, 0, NaN, an empty string, or false). If this.width is falsy, the property is not set, which can prevent potential issues or errors.